Design-Process.md

```diff
@@ -0,0 +1,194 @@
+LDT member deciding to sponsor the idea, and to bring it forward at a future LDM. Most features do not make it out of this stage.
+
+In order to move to the next stage, there needs to be enough detail to fill out the [proposal template](proposals/proposal-template.md) with at least some amount of detail.
+While we do not need exact spec language at this point, there should be enough information that other LDT members can get a general idea of the feature, what areas of the
+language it will impact, and where the complicated aspects are likely to be. In order to be triaged as part of an LDM, this template will need to be checked into the repo.
+
+#### Lifecycle
+
+* Starts when a new discussion is opened
+* Moves to [Championed feature](#championed-feature) when an LDT member decides to champion
+
+### Championed feature
+
+A championed feature is an idea for a C# language feature that an LDT member has decided to sponsor, or "champion", for possible inclusion into C#. You can identify issues
+in this category by looking for issues with
+[this query](https://github.com/dotnet/csharplang/issues?q=is%3Aissue%20state%3Aopen%20no%3Amilestone%20label%3A%22Proposal%20champion%22), issues with the `Proposal Champion`
+label and no milestone. For these issues, one or more LDT members have indicated that they are interested in the idea, but the entire LDM has not met to discuss the idea and
+give an official blessing. We try to triage these every few months, though when we start wrapping up a particular release and design time is needed for active questions on
+features currently under development, we can lag behind here.
+
+#### Lifecycle
+
+* Starts when an LDT member decides to champion a [proposed feature](#proposed-feature)
+* Moves to [rejected feature](#rejected-feature) if rejected at LDM
+* Moves to [triaged feature](#implemented-feature) if approved at LDM and assigned to a development milestone
+
+### Triaged feature
+
+A triaged feature is a championed issue that has been approved at LDM for inclusion in a future release of C#. We have quite a few issues in this bucket; they are visible
+by looking at any issues labeled `Proposal Champion` that have been assigned to one of the development milestones, `Any Time`, `Backlog`, `Needs More Work`, or `Working Set`.
+[This query](https://github.com/dotnet/csharplang/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22Proposal%20champion%22%20milestone%3ABacklog%20OR%20milestone%3A%22Any%20Time%22%20OR%20milestone%3A%22Needs%20More%20Work%22%20OR%20milestone%3A%22Working%20Set%22)
+shows these issues. The development milestones mean the following:
+
+* `Working Set` - These are features that are being actively worked on by LDT members in some form; whether that's design work behind the scenes, active LDMs discussing the
+  topics, or other actions.
```

Comment:
Is this the state of features being currently implemented by the roslyn team?

Reply:
Working Set != https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md, if that's your question.

Reply:
If it's currently being implemented, it should either be in `Working Set`, or in `Any Time`.

Reply:
Perhaps the list needs some clarification then?

`Working Set` says it should be actively worked on by *LDT members*. But if I'm just implementing it, no LDT members are working on it, right?

`Any Time` specifically only talks about being open to *community* members implementing them.

Reply:
> But if I'm just implementing it, no LDT members are working on it, right?

The LDM is reviewing design changes you're making, open questions you're asking, etc, but I can adjust the wording a bit.

As to Any Time, I wouldn't read this so precisely as spec language. All that paragraph conveys is that community members can contribute specifications and implementations. That's it. It doesn't say roslyn team members can't do that work if they so choose.